Tensile/GenerateSummations.py

```diff
@@ -155,44 +179,49 @@ def GenerateSummations(userArgs):
 
         libSolutions = tensileLibrary["solutions"]
 
-        mapper={}
+        libSolutionNames = []
         for s in libSolutions:
-            key=s["name"]
-            value=s["info"]["SolutionNameMin"]
-            mapper[key]=value
+            kenelName=s["name"]
+            libSolutionNames.append(kenelName)
 
-        data_df=pd.read_csv(dataFilePath)
-        working_data = data_df.rename(str.strip,axis='columns').rename(columns=mapper)
+        working_data=pd.read_csv(dataFilePath).rename(str.strip,axis='columns')
 
         index_keys = working_data.SizeL.unique()
         solutionsDF = working_data.filter(like='Cij')
+
         perf_max = solutionsDF.max().max().item()
-    
-        for s in solutionsForSchedule1:
-            s_state = s._state
-            solution_name = s_state["SolutionNameMin"]
-            perf_raw = working_data[solution_name] 
+
+        solutionIndex = 0
+        for s in solutionsForSchedule:
+            s_stateR = solutionStatesR[solutionIndex]
+            kenelName = libSolutionNames[solutionIndex]
```

Comment:
Spelling (kernel)

Loop variable `s` is unused. Could use
`for s_stateR, kernelName in zip(solutionStatesR, libSolutionNames):`
assuming `solutionsStatesR` and `libSolutionsNames` have the same length

Reply:
I used this version.